src/infi/systray/win32_adapter.py

```diff
@@ -80,11 +82,25 @@
     LRESULT = ctypes.c_longlong
 HANDLE = ctypes.c_void_p
 
-def convert_to_ascii(s):
+# On Win 2000 and later, tray hover text can be 128 characters, otherwise 64
+# https://msdn.microsoft.com/en-us/library/windows/desktop/bb773352%28v=vs.85%29.aspx
+# https://msdn.microsoft.com/en-us/library/ms724834%28v=vs.85%29.aspx
+if sys.getwindowsversion().major >= 5:
+    SZTIP_MAX_LENGTH = 128 
+else:
+    SZTIP_MAX_LENGTH = 64

```

Comment:
I had only looked into the differences referenced in the MSDN links, so if there are other variants that could or be considered based on the header files, let me know what they are or where to find them online and I'll see what that does to the code. We might only need a dictionary to fetch the correct `SZTIP_MAX_LENGTH` from the version.